src/client.rs

```diff
@@ -458,15 +489,27 @@ fn print_help() {
 
 //////////////////// Main /////////////////////
 fn main() {
-    drop(env_logger::init());
+    env_logger::Builder::new()
+    .format(|buf, record| {
+        writeln!(buf,
+            "{} [{:5}] - {}",
+            Local::now().format("%a %Y-%m-%d %H:%M:%S%.6f"),
+            record.level(),
+            record.args(),
+        )
+    })
+    .filter(None, LevelFilter::Trace)
+    .filter(Some("futures"), LevelFilter::Off)
+    .filter(Some("tokio_core"), LevelFilter::Off)
```

Comment:
Maybe adding another line for `mio` is a way to deal with #17?

Reply:
I tried that but it appeared to have no effect; probably didn't do it correctly so I'll try one more thing. I like your suggestion of asking on `freenode#mio`, haven't done that yet but will do soon.